README.md

```diff
@@ -71,6 +97,12 @@ will bind the following URL in your statement page
 ```
 http://www.tax.service.gov.uk/contact/accessibility-unauthenticated?service=icecreams&referrerUrl=https%3A%2F%2Fwww.tax.service.gov.uk%2Fyour-service
 ```
+
+The referrerUrl parameter should be dynamic, not hard-coded and based on the request the user made to the page
+they were visiting when clicking on the 'Accessibility statement' link. It can be constructed from the
+`platform.frontend.host` configuration key and the `request.path` as shown in the following code:
```

Comment:
Maybe a note that this is set up `app-config-common`.

Reply:
I wondered about that but then wondered because app-config-common is not a public repo, whether we should?